examples/benchmark-memory.php

```diff
@@ -38,7 +38,7 @@
 EventLoop::repeat(1, function () use (&$runs) {
```

Comment:
This should probably be 0.001 then?

Reply:
Probably yes, thank you. I've committed your suggested change.